src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? getPeopleBySex('m', people).filter(person => (
+      century === Math.ceil(person.died / 100)
+    ))
+    : getPeopleBySex('m', people);
```

Comment:
I guess, it would be better to move  `getPeopleBySex('m', people)` out of ternary operator, not to repeat the same. Like this:
 ```suggestion
    ? .filter(person => (
      century === Math.ceil(person.died / 100)
    ))
    : people;
    men = getPeopleBySex('m', people);
```

Reply:
Yeah, thanks!) It makes sense)

Reply:
agree 100%